src/main/java/com/iabtcf/ByteBitVector.java

```diff
@@ -63,6 +68,10 @@ private boolean ensureReadable(int offset, int length) {
             return true;
         }
 
+        if (is == null) {
```

Comment:
Is this the best way to handle null because this class has constructor `public ByteBitVector(byte[] buffer) ` which always define field `is` as null  and is used in multiple places?

Reply:
what's the issue here? null is an acceptable value for is

Reply:
oh i see your concern..

Reply:
yes, this is fine. `is` is only null when we've passed passed the byte array. this method attempts to read additional data from the inputstream to satisify the offset + length amount of data the user has requested to be read. if the input stream is null, it's clear there won't be any data additional data to read from the input stream.

when using the other form of the constructor, the input stream will be non-null and we'll fail in the read() method if there is no additional data to read.

either way, we'll throw a byteparseexception.